applications/cm-service/templates/deployment.yaml

```diff
@@ -40,22 +61,32 @@ spec:
               path: "/"
               port: "http"
           resources:
-            {{- toYaml .Values.resources | nindent 12 }}
+            {{- toYaml .Values.frontend.resources | nindent 12 }}
           securityContext:
             allowPrivilegeEscalation: false
             capabilities:
               drop:
                 - "all"
             readOnlyRootFilesystem: true
-      {{- with .Values.nodeSelector }}
+          volumeMounts:
+            - mountPath: "/output"
+              name: "output-volume"
+              {{- with .Values.config.outputVolume.subPath }}
+              subPath: {{ . }}
+              {{- end }}
+      volumes:
+        - name: output-volume
+          persistentVolumeClaim:
+            claimName: cm-service-output
+      {{- with .Values.frontend.nodeSelector }}
       nodeSelector:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      {{- with .Values.tolerations }}
+      {{- with .Values.frontend.tolerations }}
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       securityContext:
         runAsNonRoot: true
-        runAsUser: 1000
-        runAsGroup: 1000
+        runAsUser: 17951
```

Comment:
Break out to values file?

Reply:
Good point -- USDF (or other site) specific -- will do.

Reply:
Actually, upon inspection, this user/group is already specified as the default in the container image.  I will just remove these.